contracts/governance/utils/VotesExtended.sol

```diff
@@ -34,8 +34,8 @@ abstract contract VotesExtended is Votes {
     using Checkpoints for Checkpoints.Trace160;
     using Checkpoints for Checkpoints.Trace208;
 
-    mapping(address delegator => Checkpoints.Trace160) private _delegateCheckpoints;
-    mapping(address account => Checkpoints.Trace208) private _balanceOfCheckpoints;
+    mapping(address delegator => Checkpoints.Trace160) private _userDelegationCheckpoints;
+    mapping(address account => Checkpoints.Trace208) private _userVotingWeightCheckpoints;
```

Comment:
It's not clear what is represented here based on the name--delegated voting weight (the default way of referencing voting weight), or the "intrinsic" voting weight to an address.